R/plotSeries.R

```diff
@@ -301,9 +300,10 @@ setMethod("plotSeries", signature = c(object = "SummarizedExperiment"),
                                 linetype_by = linetype_by,
                                 size_by = NULL,
                                 alpha = line_alpha,
+                                linetype = line_type,
                                 size = line_size)
     
-    line_args$args$mapping$group <- sym("colour_by")
+    #line_args$args$mapping$group <- sym("colour_by")
```

Comment:
Now, when I commented row 306, it is working. I do not know, if grouping is necessary. I think it is done automatically.

Reply:
I am not sure. sometimes it, and sometimes it is not. I think you have to group based on the `series` identifier aka. the rowname, which the date to plot was derived from.

I think your are adding this to the `sample` column in `.melt_series_data`. It probably would be better named `feature` (Line 279 in the current state)